va/validation-authority.go

```diff
@@ -208,6 +237,15 @@ func (va ValidationAuthorityImpl) validateDvsni(identifier core.AcmeIdentifier,
 	z := sha256.Sum256(RS)
 	zName := fmt.Sprintf("%064x.acme.invalid", z)
 
+	// Check for resolver SERVFAIL for A/AAAA records
```

Comment:
Well, #470 is sort of a feature bug, whereas the rest of this is test improvement.